tests/utilities/test_opt_variables.py

```diff
@@ -84,117 +86,83 @@ def test_adjust_fixed(self):
-
-        opt_vars.adjust_variables(
+        self.vars.adjust_variables(
             {"a": {"value": -2, "lower_bound": -1, "upper_bound": 3}},
             strict_bounds=False,
         )
-        assert opt_vars["a"].value == -2
-        assert opt_vars["a"].lower_bound == -2
+        assert self.vars.a.value == -2
+        assert self.vars.a.lower_bound == -2
 
         with pytest.raises(OptVariablesError):
-            opt_vars.adjust_variables(
+            self.vars.adjust_variables(
                 {"a": {"value": -2, "lower_bound": -1, "upper_bound": -3}},
-                strict_bounds=False,
+                strict_bounds=True,
             )
 
     def test_read_write(self):
         tempdir = tempfile.mkdtemp()
         try:
             the_path = os.sep.join([tempdir, "opt_var_test.json"])
             self.vars.to_json(the_path)
-            new_vars = OptVariables.from_json(the_path)
+            new_vars = TestOptVariablesOptVariables.from_json(the_path)
             assert new_vars.as_dict() == self.vars.as_dict()
         finally:
             shutil.rmtree(tempdir)
 
     def test_tabulate_displays_column_values(self):
-        v1 = BoundedVariable("a", 2, 0, 3, descr="var a")
-        v2 = BoundedVariable("b", 0, -1, 1, fixed=True)
-        v3 = BoundedVariable("c", -1, -10, 10)
-        opt_vars = OptVariables([v3, v1, v2])
-
-        table = opt_vars.tabulate()
+        table = self.vars.tabulate()
 
         table_pattern = "\n".join(
             [
-                ".*",
-                ".*Name.*Value.*Lower Bound.*Upper Bound.*Fixed.*Description.*",
-                ".*",
-                ".*a.*2.*0.*3.*False.*var a.*",
-                ".*",
-                ".*b.*0.*-1.*1.*True.* .*",
-                ".*",
-                ".*c.*-1.*-10.*10.*False.* .*",
-                ".*",
+                "TestOptVariablesOptVariables",
+                "╒════════╤═════════╤═══════════════╤═══════════════╤═════════╤═══════════════╕",
+                "│ name   │   value │   lower_bound │   upper_bound │ fixed   │ description   │",
+                "╞════════╪═════════╪═══════════════╪═══════════════╪═════════╪═══════════════╡",
+                "│ a      │       2 │             0 │             3 │ False   │               │",
+                "├────────┼─────────┼───────────────┼───────────────┼─────────┼───────────────┤",
+                "│ b      │       0 │            -1 │             1 │ False   │               │",
+                "├────────┼─────────┼───────────────┼───────────────┼─────────┼───────────────┤",
+                "│ c      │      -1 │           -10 │            10 │ False   │               │",
+                "╘════════╧═════════╧═══════════════╧═══════════════╧═════════╧═══════════════╛",
             ]
         )
-        assert len(table.split("\n")) == 9
+        assert len(table.split("\n")) == 10
         assert re.match(table_pattern, table, flags=re.MULTILINE)
```

Comment:
Is this not now just string comparison rather than the regex that it was before (which looks like it should still work?)